datalad/support/annexrepo.py

```diff
@@ -2636,7 +2655,10 @@ def repo_info(self, fast=False):
 
         options = ['--bytes', '--fast'] if fast else ['--bytes']
 
-        json_records = list(self._run_annex_command_json('info', opts=options))
+        # note - shouldn't use merge_annex_branches=False because updated
+        # info/description about remote UUIDs might be needed
+        json_records = list(self._run_annex_command_json(
```

Comment:
OK, but this does mean that the `datalad ls -L` usecase isn't resolved by this PR, right?

Reply:
I thought it was even after my fix but I could be wrong... The test will show. Thanks for catching this!